video-processing-service/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "video-processing-service",
     "version": "1.0.0",
     "type": "module",
-    "main": "index.js",
+    "main": "dist/index.js",
```

Comment:
`nit` `suggestion`:  It would be good developer ergonomics to have this `build` and then start the `dev` server. Because whenever we make a change, I want to see a hot reload of the built code. So it would be nice to have a command that builds this like it would in prod. But don't worry, you'll see this down the line.